src/components/AccountModule/ClientSyncedInfo.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { GU } from '@aragon/ui'
+import { useClientBlockNumber } from './useClientBlockNumber'
+import { getClientSyncState } from './utils'
+
+function ClientSyncedInfo({ listening, online, syncDelay }) {
+  const latestClientBlockNumber = useClientBlockNumber()
+  const { header, info } = getClientSyncState(
+    listening,
+    online,
+    syncDelay,
+    latestClientBlockNumber
+  )
+
+  return (
+    <React.Fragment>
+      {header && info && (
+        <div
+          css={`
+            margin-top: ${1 * GU}px;
+          `}
+        >
+          <span
+            css={`
+              padding-right: ${1 * GU}px;
+              opacity: 0.8;
```

Comment:
We generally avoid using `opacity` to change the colour; we should have a `surfaceSecondary` colour that fits here.

Reply:
Aaah yes, this is actually some leftover from the old account module. :)